src/components/command-bar/commands/search/components/dialog-body/index.tsx

```diff
@@ -152,7 +155,17 @@ const SearchCommandBarDialogBodyContent = ({
 					const { heading, icon, content } =
 						tabsBySearchableContentType[contentType]
 					return (
-						<Tab heading={heading} icon={icon} key={contentType}>
+						<Tab
+							heading={heading}
```

Comment:
Do we still need to pass heading if the heading slot is being used?

Reply:
Unfortunately yes, as `headingSlot` doesn't work for the `<select />`-based Tabs interface used on mobile. I'll add a note in the `types` file for `heading` to make this more clear 👍

Reply:
✅ I've added comments to `types` files to clarify the relationship between `heading` and `headingSlot` (which ultimately become `label` and `labelSlot`, bit of a historical naming conflict I think).